### PR TITLE
Added signature invalid error page

### LINKED ISSUE #194: Add signature invalid error page
Add the signature invalid error page, the current stock one looks like this
image

### PR DESCRIPTION
**Pull request by MEDANIS98**
*Added `403.blade.php` to `errors/views/resouress`
*closes #194 
Signed-off-by: Sadoudi Anis <sadoudimohamedanis@gmail.com>

### CODE REVIEW

File: resources/lang/fr.json [hunk 1]
```diff
@@ -152,5 +152,6 @@
 	"Add to wish list": "Ajouter à la liste de souhaits",
 	"Phone number": "Numéro de Teléphone",
 	"Call Us": "Appelez-nous",
-	"Page has expired, try refreshing the page": "La page a expiré, essayez d'actualiser la page"
-}
\ No newline at end of file
+	"Page has expired, try refreshing the page": "La page a expiré, essayez d'actualiser la page",
+	"Access Forbidden": "accès interdit"
```
Comment (CaddyDz) [inline resources/lang/fr.json:156]: Capitalize the localized strings

File: resources/views/errors/403.blade.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@extends('layouts.app')
+
+@section('title', __('Access Forbidden'))
+
```
Comment (CaddyDz) [inline resources/views/errors/403.blade.php:4]: Remove empty line

File: resources/views/partials/account_menu.blade.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <div class="account-menu__divider"></div>
     <a href="#" class="account-menu__user">
         <div class="account-menu__user-avatar">
-            <img src="{{ auth()->user()->accountMenuAvatar }}" alt="@lang('Avatar')">
+            {{-- <img src="{{ auth()->user()->accountMenuAvatar }}" alt="@lang('Avatar')"> --}}
```
Comment (CaddyDz) [inline resources/views/partials/account_menu.blade.php:6]: Don't push this

File: resources/views/errors/403.blade.php (hunk 2 above)
Comment (MEDANIS98) [reply resources/views/errors/403.blade.php:4]: done

File: resources/views/partials/account_menu.blade.php (hunk 3 above)
Comment (MEDANIS98) [reply resources/views/partials/account_menu.blade.php:6]: okay

**Pull request merged**